lsp-ui-imenu.el

```diff
@@ -76,6 +92,9 @@
 (declare-function imenu--subalist-p 'imenu)
 (defvar imenu--index-alist)
 
+(defvar lsp-ui-imenu--refresh-timer nil
```

Comment:
This should be a local variable, I think ?

Reply:
There is only one imenu window right? If this is true, the I don't see the benefit setting it to local.

Reply:
You can have multiple frames of emacs open, so there can be more than 1 imenu window

Reply:
Got you! Fixed! 👍